marklogic-data-hub/src/main/java/com/marklogic/hub/cli/command/InstallIntoDhsCommand.java

```diff
@@ -120,8 +120,6 @@ public void applyDhsSpecificProperties(Properties props, boolean disableSsl) {
 
         props.setProperty("mlFlowDeveloperRole", "flowDeveloper");
         props.setProperty("mlFlowOperatorRole", "flowOperator");
-        props.setProperty("mlModulePermissions",
-            "flowDeveloper,read,flowDeveloper,execute,flowDeveloper,insert,flowOperator,read,flowOperator,execute,flowOperator,insert");
```

Comment:
I think this needs to remain for now, as these are defined via DHS-specific roles.